packages/driver/src/dom/coordinates.js

```diff
@@ -203,23 +203,30 @@ const getBottomRightCoordinates = (rect) => {
 const getElementCoordinatesByPositionRelativeToXY = ($el, x, y) => {
   const positionProps = getElementPositioning($el)
 
-  const { fromElViewport, fromElWindow } = positionProps
+  const { fromElViewport, fromElWindow, fromAutWindow } = positionProps
 
   fromElViewport.left += x
   fromElViewport.top += y
 
   fromElWindow.left += x
   fromElWindow.top += y
 
+  fromAutWindow.left += x
+  fromAutWindow.top += y
+
   const viewportTargetCoords = getTopLeftCoordinates(fromElViewport)
   const windowTargetCoords = getTopLeftCoordinates(fromElWindow)
+  const AUTwindowTargetCoords = getTopLeftCoordinates(fromAutWindow)
```

Comment:
this variable should really be named `autWindowTargetCoords` and not be capitalized, but ill approve anyway and fix it in develop